.github/workflows/release-staging.yaml

```diff
@@ -0,0 +1,129 @@
+name: "ODH Release - Staging Pipeline"
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        type: string
+        description: The version to update(https://semver.org/).
+        required: true
+      tracker-url:
+        type: string
+        description: The URL to tracker issue(https://github.com/opendatahub-io/opendatahub-community/issues/<issue-number>).
+        required: true
+env:
+  VERSION: ${{ github.event.inputs.version }}
+  TRACKER_URL: ${{ github.event.inputs.tracker-url }}
+jobs:
+  stage-release:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+      pull-requests: write
+    steps:
+    - uses: actions/checkout@v4
+
+    - name: Validate tracker URL
+      run: |
+        if ! echo "${{ env.TRACKER_URL }}" | grep -qE '^https://github\.com/opendatahub-io/opendatahub-community/issues/'; then
+          echo "Error: Invalid tracker URL format. Expected format: https://github.com/opendatahub-io/opendatahub-community/issues/<issue-number>"
+          exit 1
+        fi
+
+    - name: Extract issue number
+      run: |
+        ISSUE_NUMBER=$(echo "${{ env.TRACKER_URL }}" | grep -oE '[0-9]+$')
+        echo "ISSUE_NUMBER=$ISSUE_NUMBER" >> $GITHUB_ENV
+
+    - name: Validate semver
+      run: ./.github/scripts/validate-semver.sh v${{ env.VERSION }}
+
+    - uses: fregante/setup-git-user@v2
+
+    - name: Create release branch
+      run: |
+        # Check if branch exists remotely - handle the case if we are retriggering the release workflow after a failure in the middle
+        if git ls-remote --heads origin odh-${{ env.VERSION }} | grep -q "odh-${{ env.VERSION }}"; then
+          echo "Branch odh-${{ env.VERSION }} already exists, checking it out"
+          git fetch origin odh-${{ env.VERSION }}
+          git checkout odh-${{ env.VERSION }}
+          git pull origin odh-${{ env.VERSION }}
+        else
+          echo "Creating new branch odh-${{ env.VERSION }}"
+          git checkout -b odh-${{ env.VERSION }} origin/main
+          git push -u origin odh-${{ env.VERSION }}
+        fi
+
+    - name: Update versions in relevant files
+      run: ./.github/scripts/update-versions.sh ${{ env.VERSION }}
+
+    - name: Run make manifests & bundle
+      run: make manifests bundle
+
+    - name: Clean up bundle files
+      run: |
+        sed -i -e "s|image: quay.io/opendatahub/opendatahub-operator:latest.*|image: REPLACE_IMAGE:latest|g" bundle/manifests/opendatahub-operator.clusterserviceversion.yaml
+        rm -f ./config/manager/kustomization.yaml
+
+    - name: Update manifest branches
+      uses: ./.github/actions/update-manifest-branches
```

Comment:
we do not need action here right?
also that action is "Create release pr" which we do not need any more
just put 
```
- name: Update versions
      shell: bash
      run: ./.github/scripts/update-manifests-tags.sh
```
here , enought?

Reply:
Yes, I missed that. Previous workflow needed it in 2 places, now it is used in only one place so no need to  use a reusable logic